app/assets/stylesheets/accounts.scss

```diff
@@ -44,6 +44,15 @@
     }
   }
 
+  .joined-at {
+    float:left;
+    padding-left: 10px;
+
+    &:hover {
```

Comment:
I changed my mind about a tooltip and apparently forgot to remove the hover, I'll change that.